.github/workflows/standard.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       # Extras that will be installed for both conda/pip:
       testing-extras: "happi"
       # Extras to be installed only for conda-based testing:
-      conda-testing-extras: "pcdsdevices pyqt=5"
+      conda-testing-extras: "pyqt=5"
       # Extras to be installed only for pip-based testing:
       pip-testing-extras: "PyQt5<5.15"
```

Comment:
We should make sure this is in `requirements.txt` for pip-based installs (`PyQt5` no pin)

Reply:
`PyQt5` is in `requirements.txt` 👍

Reply:
```suggestion
      pip-testing-extras: ""
```